onnx/backend/base.py

```diff
@@ -100,44 +97,34 @@ def prepare(
 
     @classmethod
     def run_model(
-        cls, model: ModelProto, inputs: Any, device: str = "CPU", **kwargs: Any
+        cls, model: ModelProto, inputs: Any, *args, **kwargs: Any
     ) -> tuple[Any, ...]:
-        backend = cls.prepare(model, device, **kwargs)
+        backend = cls.prepare(model, *args, **kwargs)
         assert backend is not None
         return backend.run(inputs)
 
     @classmethod
     def run_node(
         cls,
         node: NodeProto,
-        inputs: Any,  # noqa: ARG003
-        device: str = "CPU",  # noqa: ARG003
```

Comment:
These are public apis in this file we cannot change without carefully considering depreciation and migration